wgpu-core/src/command/transfer.rs

```diff
@@ -198,17 +203,13 @@ pub(crate) fn validate_texture_copy_range(
         texture_copy_view
             .mip_level
             .try_into()
-            .expect("Mip level must be < 256"),
+            .map_err(|_| TransferError::InvalidMipLevel)?,
```

Comment:
I don't think we need this error. It should be caught by the check that the texture *has* this mipmap level. Obviously, it can't have anything above 32